packages/composite-checkout-wpcom/src/components/wp-contact-form.js

```diff
@@ -89,11 +84,13 @@ const FieldRow = styled( GridRow )`
 	}
 `;
 
-function isElligibleForVat() {
+function isEligibleForVat( country ) {
 	//TODO: Detect whether people are in EU or AU and return true if they are
-	return false;
+	const countriesWithVAT = [ 'AU' ];
+	return countriesWithVAT.includes( country );
 }
 
+// TODO: Figure out if we still need this
```

Comment:
No, you're right. This should be removed. The linters didn't find it because of its proptypes definition. It would make sense to remove it here. Thanks!